projects/plugins/jetpack/extensions/blocks/story/player/lib/wait-media-ready.js

```diff
@@ -22,18 +23,14 @@ export default async function waitMediaReady( mediaElement, fullLoad = false ) {
 			if ( mediaElement.type ) {
 				requestHeaders.append( 'Content-Type', mediaElement.type );
 			}
-			return fetch( videoRequest, {
+			const response = await fetch( videoRequest, {
```

Comment:
I'm back to seeing endless loading animations on video-first stories a lot (https://github.com/Automattic/story-block/issues/12), I guess it might be because of this change?

Reply:
Nevermind, I'm not able to reproduce this anymore, and per our conversation this shouldn't have affected the poster image loading.